IronPython3Tests/IronPython3Tests.csproj

```diff
@@ -0,0 +1,34 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>net8.0</TargetFramework>
+    <ImplicitUsings>enable</ImplicitUsings>
+    <Nullable>enable</Nullable>
+
+    <IsPackable>false</IsPackable>
+    <IsTestProject>true</IsTestProject>
+  </PropertyGroup>
+
+  <ItemGroup>
+	  <PackageReference Include="DynamoVisualProgramming.Tests" Version="3.1.0-beta3874">
+		  <PrivateAssets>all</PrivateAssets>
```

Comment:
why PrivateAssets all ?

Reply:
just usually the first thing I try, and if the tests run fine without any deps copied then great!

Reply:
I do not think it is useful in a test project
From the documentation  https://learn.microsoft.com/en-us/nuget/consume-packages/package-references-in-project-files#controlling-dependency-assets :

```
PrivateAssets | These assets will be consumed but won't flow to the parent project
```

If you want to have assemblies/content that is compile time only, you should use IncludeAssets/ExcludeAssets

Reply:
I think the assmblies in the dynamo nuget should be copied. THey are needed to actually discover/run the tests

Reply:
actually... I think it doesn't matter - I got confused with `<ExcludeAssets> - I don't think anyone will consume this particular project so it should not matter.` I will remove it.